### ISSUE TITLE
Why does the loss decrease during training, but the PSNR value gets lower?

### ISSUE THREAD

**Issue by Joeytime** [csbhr/Self-Blind-VSR#6]
(1) I replaced different training sets, and this problem always occurred during training. The loss has been decreasing, but the highest PSNR value is often the beginning epoch.

![WechatIMG1811](https://github.com/user-attachments/assets/75e94590-d919-4f1c-9db7-717a689ad73c)
![WechatIMG1812](https://github.com/user-attachments/assets/b87488b3-5311-40fa-a2a4-ef84726ff5f7)

(2) Why is the input for training here the '../dataset/REDS/train/HR' folder?
    |--train  
        |--HR  
            |--video 1
                |--frame 1
                |--frame 2
                    ：  
            |--video 2
                :
            |--video n
            
def set_template(args):
    if args.template == 'Self_Blind_VSR_Gaussian':
        args.task = "FlowVideoSR"
        args.model = "PWC_Recons"
        args.save = "Self_Blind_VSR_Gaussian"
        args.data_train = 'REDS_BLURDOWN_GAUSSIAN'
        args.dir_data = '../dataset/REDS/train'
        args.data_test = 'REDS_HRLR'
        args.dir_data_test = '../dataset/REDS4_BlurDown_Gaussian'
        args.HR_in = False
        args.scale = 4
        args.patch_size = 160
        args.n_sequence = 5
        args.n_frames_per_video = 50
        args.n_feat = 128
        args.extra_RBS = 3
        args.recons_RBS = 20
        args.ksize = 13
        args.loss = '1*L1'
        args.lr = 1e-4
        args.lr_decay = 100
        args.save_middle_models = True
        args.save_images = False
        args.epochs = 500
        args.batch_size = 8
If I replace it with LR, there will be a patch size problem.

**Comment by qqhe-frank** [comment]
I having the same first problem where the losses are going down, but the metrics are going down just as much. I guessing the reason for this is that the reconstruction network doesn't really learn during the training process, but rather amounts to a residual shortcut that leads to worse and worse PSNR results.

**Comment by SaeedNowrouzian** [comment]
I am having trouble installing and running this code. If possible, friends who have successfully implemented the code. Write the requirements.txt file and type of operating system.
